Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM  python:3  
+FROM  python:3
 
-RUN pip install fakelogs
+COPY . /tmp/fakelogs
+RUN pip install /tmp/fakelogs && \
+    rm -r /tmp/fakelogs
+
+#RUN pip install fakelogs
```

Comment:
So, idk about this change being "the right way to do it", but it worked so I could test the docker image.

Reply:
yeah.  I think this is better since it builds the version on disk instead of the most recent version pushed to pypi